kmath-ast/src/jvmMain/kotlin/scientifik/kmath/asm/Optimization.kt

```diff
@@ -0,0 +1,48 @@
+package scientifik.kmath.asm
```

Comment:
I suggest to bundle all misc methodsin one file or create `internal` directory for them to limit their visibility.

Reply:
OK

Reply:
OK